docs/source/noise/index.rst

```diff
@@ -93,6 +93,108 @@ column shows the name of the noise type in the configuration system.
 Default Noise Types for Each Column
 -----------------------------------
 
+.. list-table:: Types of noise for each column
+  :widths: 20 20 20 20
+  :header-rows: 0
+
+  * - Attribute name
+    - Datasets present
+    - Types of noise
+    - Notes
+  * - First name
+    - Census, Household Surveys, WIC, Taxes (both), SSA
```

Comment:
Can you match the Dataset names to what they are called on the Datasets page, using abbreviations? And remove 1040 since that is not implemented. E.g. this line would be

```suggestion
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
```